restore-sql-server-transaction-logs/scheduled-upload/upload-script.ps1

```diff
@@ -0,0 +1,46 @@
+$settingsFile="<full-path-to-your-script>\settings.json"
```

Comment:
This file should have a well-defined path instead of being configurable. Otherwise, you need to instruct users about how to edit this line, but then you'd be leaking configuration in a file (this one) that's not a config file.

Reply:
Removed the setting file.